utils/docker/Makefile

```diff
@@ -37,11 +37,19 @@ base:
 ccenv: base
 	$(DOCKER) build -t $(FPC_DOCKER_NAMESPACE)-ccenv ccenv
 
+# Note: for overall consistency reasons we want the FPC code from the current repo
+# as this will rebuild each time and take a while, we don't add a dependency
+# above in build ...
```

Comment:
Which dependency are you referring to here?

Reply:
i meant a dependency to the build target. Rephrased now ...